plugins/cue_points/base/batch/copyCuePoints/Engine/KLiveClippingCopyCuePointEngine.php

```diff
@@ -24,4 +24,14 @@ public function getCuePointFilter($entryId, $status = CuePointStatus::READY)
         $filter = parent::getCuePointFilter($entryId, implode(",",$statuses));
         return $filter;
     }
+    
+    /**
+     * @param KalturaCuePoint $cuePoint
+     * @return boolean
+     */

```

Comment:
Function rename checkShouldCopyCuePointBeforeTimeWindow -> shouldCopyCuePointBeforeTimeWindow
or shouldCopyBasedOnTags and just send it the tags list, also - is it relevant to all cue point types or only to a specific type, if it is relevant to a specific type then I will set this function only on the relevant cue point object.